hack/lima-kubestellar.yaml

```diff
@@ -0,0 +1,179 @@
+# Please edit the following configuration for Lima instance "kubestellar"
+# and an empty file will abort the edit.
+
+# A template to install and use a KubeStellar demo environment with Lima.
+# $ limactl start lima-kubestellar.yaml --name kubestellar --vm-type=vz --mount-type=virtiofs --cpus 6 --memory 32 --tty=false
+# $ limactl shell kubectl --context its1 get managedclusters
+
+# To access KubeStellar on the host (assumes kubectl is installed), run the following commands:
+# $ TODO - need something to interact with kubestellar from the host... maybe a kflex command?
+
+# This template requires Lima v0.8.0 or later
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release-20241119/ubuntu-24.04-server-cloudimg-amd64.img"
```

Comment:
FYI, it looks like this is the last daily build of that release, which equals the non-daily release image below byte-for-byte.